yellowbrick/__init__.py

```diff
@@ -1,10 +1,10 @@
 # yellowbrick
 # A suite of visual analysis and diagnostic tools for machine learning.
 #
-# Author:   Rebecca Bilbro <rbilbro@districtdatalabs.com>
+# Author:   Rebecca Bilbro
```

Comment:
@rebeccabilbro if possible, I'd like to add my name here as well

Reply:
absolutely!